IopHomeNode.proto3

```diff
@@ -433,12 +433,12 @@ message ServerRole {
 //      |                                                                     |
 //      |  StartConversationRequest                                           |
 //      *-------------------------------------------------------------------> *
-//      |                                                                     |
+//      |  <clientChallenge>                                                  |
 //      |                                                                     | status := ConversationStarted
 //      |                                                                     |
 //      |                                          StartConversationResponse  |
 //      * <-------------------------------------------------------------------*
-//      |                                                        <challenge>  |
+//      |                                <challenge, signed clientChallenge>  |
```

Comment:
Being pedantic: I thought signature is to the whole payload of response, including challenge and clientChallenge?